engines/scumm/he/wiz_he.cpp

```diff
@@ -1491,7 +1492,29 @@ uint8 *Wiz::drawWizImage(int resNum, int state, int maskNum, int maskState, int
 	}
 
 	if (flags & kWIFPrint) {
-		error("WizImage printing is unimplemented");
+		switch (comp) {
+		case 0: {
+			Graphics::ManagedSurface surf;
+			surf.w = width;
+			surf.h = height;
+			surf.format=Graphics::PixelFormat::createFormatCLUT8();
+
+			uint8 *wizd = _vm->findWrappedBlock(MKTAG('W', 'I', 'Z', 'D'), dataPtr, state, 0);
+			assert(wizd);
+			surf.setPixels(wizd);
+
+			uint8 *pal = _vm->findWrappedBlock(MKTAG('R', 'G', 'B', 'S'), dataPtr, state, 0);
+			assert(pal);
+			surf.setPalette(pal, 0, 256);
+
+			PrintingManager *pm = _vm->_system->getPrintingManager();
+			pm->printImage(surf);
+
+			return 0;
+		}
+		default:
+			error("Unsupported compression mode %d", comp);
```

Comment:
Let's talk about the  compmode 1 here.

Probably, you do not need to understand it, just call those unpackers here and they will return you a normal bitmap. E.g. just copy over the calls and checks for masks.